libfreerdp/codec/rfx_rlgr.c

```diff
@@ -101,7 +103,9 @@ static INLINE UINT32 lzcnt_s(UINT32 x)
 		return n - x;
 	}
 
+#ifndef __MINGW32__
```

Comment:
Why this? If the symbol is not native, it should be replaced by WinPR version of it.

Reply:
Right, will fix.